cdk/src/cdk.ts

```diff
@@ -13,9 +15,9 @@ new OpenDataPlatformStack(app, 'CdkStack', {
    * and Region that are implied by the current CLI configuration. */
   // env: { account: process.env.CDK_DEFAULT_ACCOUNT, region: process.env.CDK_DEFAULT_REGION },
 
-  /* Uncomment the next line if you know exactly what Account and Region you
-   * want to deploy the stack to. */
-  // env: { account: '123456789012', region: 'us-east-1' },
+  // Development account.
+  // TODO: replace with AWSGoogleSandbox account.
```

Comment:
Is this just for default config or do we generally want to move to deploying to AWSGoogleSandbox? My understanding was that we would be deploying the actual project to BlueConduit's accounts and that the sandbox account was just for us to experiment with cdk tutorials.

Reply:
That's a good question that I don't know the answer to. I'm going to leave this comment in there temporarily until we figure that out, but I want to submit this in the meantime.